wai-extra/Network/Wai/Middleware/RequestLogger.hs

```diff
@@ -50,14 +51,37 @@ import Network.Wai.Middleware.RequestLogger.Internal
 import Network.Wai.Header (contentLength)
 import Data.Text.Encoding (decodeUtf8')
 
+-- | The logging format.
 data OutputFormat
   = Apache IPAddrSource
-  | Detailed Bool -- ^ use colors?
+  | Detailed DetailedSettings
```

Comment:
Isn't this backwards incompatible? And do we want to conserve backwards compatibility? 
I think this is a rather common package, and it might break things on a bunch of packages. 
I'd therefore propose that we add a separate data constructor for our `DetailedSettings` based logger.

Reply:
Yeah, I see.